### PR TITLE
Fix V2V inital state

### PR DESCRIPTION
**Pull request by mareklibra**

### CODE REVIEW

**Comment by mareklibra** [comment]
v2vStatus png

**Comment by mareklibra** [comment]
@andybraren , any idea for better state naming?
Recent `Importing (VMWare) Pending` or  `Importing (VMWare) Error` is clumsy.

**Comment by suomiy** [comment]
reminder: this is relevant to our discussion in https://github.com/kubevirt/web-ui-components/pull/427

**Comment by andybraren** [comment]
> any idea for better state naming? Recent Importing (VMWare) Pending or Importing (VMWare) Error is clumsy.

I agree with @suomiy's suggested labeling scheme:

- Import Pending (VMware)
- Importing (VMware)
- Import Error (VMware)

As discussed in #427, if the "(VMware)" part starts getting cut off with "..." that's fine. It's the least important part of the status message, and clicking the status should reveal a popover like the one shown [here](https://github.com/kubevirt/web-ui-components/pull/427#issuecomment-490125250) in #427.

FYI @matthewcarleton ^

57310568-057ecc00-70b8-11e9-98a1-5181b8b7f724

**Comment by mareklibra** [comment]
Statuses rephrased.

File: src/components/VmStatus/VmStatus.js [hunk 1]
```diff
@@ -54,14 +54,19 @@ const StateV2VConversionInProgress = ({ progress, ...props }) => (
     Importing (VMWare)
```
[the remark this answers is not in the document]
Comment (suomiy) [reply src/components/VmStatus/VmStatus.js]: @mareklibra ah I just noticed - the name is still incorrect here.  Can you please do a followup?

**Comment by mareklibra** [comment]
@suomiy , https://github.com/kubevirt/web-ui-components/pull/455

**Pull request merged**